docs/topics/signing.txt

```diff
@@ -25,15 +25,19 @@ You may also find signing useful for the following:
   protected resource, for example a downloadable file that a user has
   paid for.
 
-Protecting the ``SECRET_KEY``
-=============================
+Protecting ``SECRET_KEY`` and ``SECRET_KEY_FALLBACKS``
+======================================================
 
 When you create a new Django project using :djadmin:`startproject`, the
 ``settings.py`` file is generated automatically and gets a random
 :setting:`SECRET_KEY` value. This value is the key to securing signed
 data -- it is vital you keep this secure, or attackers could use it to
 generate their own signed values.
 
+:setting:`SECRET_KEY_FALLBACKS` can be used to rotate secret keys. The
+values will not be used to sign data, but if specified, they will be used to
+validate signed data and must be kept secure.
+
```

Comment:
```suggestion
.. versionchanged:: 4.1

    The `SECRET_KEY_FALLBACKS` setting was added.
```

Reply:
@felixxm The lack of spacing around the annotation seems to be different than the rest of the documentation. Is this correct?

Reply:
@felixxm I had to revert this suggestion in order to please the spell checker.

Reply:
> @felixxm The lack of spacing around the annotation seems to be different than the rest of the documentation. Is this correct?

Yes, a blank line should be there. Adding suggestion via GitHub is not always easy, typos can happen.

> @felixxm I had to revert this suggestion in order to please the spell checker.

Why? :thinking: Maybe because I missed double backticks:
```
.. versionchanged:: 4.1

    The ``SECRET_KEY_FALLBACKS`` setting was added.
```

Reply:
WIthout the leading blank link, it was complaining that there was a misspelling in `.. versionchanged:: 4.1`. I'm not sure if it was the `..`, `versionchanged` or `::`. However, adding the leading blank line got it to recognize it as sphinx syntax and ignore the line entirely.